sdk/src/main/java/io/radar/sdk/RadarJobScheduler.kt

```diff
@@ -134,11 +134,11 @@ class RadarJobScheduler : JobService() {
             }
 
             val source = extras.getString(EXTRA_SOURCE)
-            //this may occur, for example, if a new location job is scheduled before the current job finishes executing.
-            Radar.logger.d("Stopping Location Job | location = $location; source = $source")
+            // This may occur, for example, if a new location job is scheduled before the current job finishes executing.
+            Radar.logger.d("Stopping location job | location = $location; source = $source")
```

Comment:
@jsani-radar can we add a write-up, or maybe start a spec for these standards? Nothing stopping these changes going in, just want to be able to add some automated checks for these for next time (or elsewhere). The refactor PR adds a bunch of static analysis checks, and these could be done as part of that.

Reply:
yeah good call - it's been an AI in my backlog to create an SDK style guide with conventions for comments / release notes / log messages, on me to prioritize that